server/server.go

```diff
@@ -63,6 +63,13 @@ var (
 	ErrNoSession = status.Errorf(codes.Unauthenticated, "no session information")
 )
 
+var noCacheHeaders = map[string]string{
+	"Expires":         time.Unix(0, 0).Format(time.RFC1123),
+	"Cache-Control":   "no-cache, private, max-age=0",
```

Comment:
If we're trying to disable caching entirely, wondering if the [MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control#Preventing_caching) suggestion of `Cache-Control: no-cache, no-store, must-revalidate` might avoid potential edge cases, since `private` allows caching in private (end-user) browser caches.  In the end, it probably doesn't matter.  Looking good!